mafiasi/settings.py.example

```diff
@@ -199,3 +199,5 @@ USER_LOGIN_HINT = "Note: For our account names we use two digits for year (e.g.
 GUEST_INVITE_INSTRUCTION_LINK = "https://dash.crossmodal-learning.org/static_redir/instruction"
 
 GUEST_ACCEPT_INVITATION_MAIL = False
+
+USERNAME_PATTERN = r'\d?[a-z]+'
```

Comment:
Given that we actually match this against the mailbox name upon registration rather than the actual (later) username, I would not call this `USERNAME_PATTERN`.

If this was to be used for our usernames, it would rather need to be something like `(\d{2})?` or possibly a more lenient `\d{0,2}`.

Reply:
You are right. I changed it to `ACCOUNT_PATTERN`. Our usernames are explicitly not allowed there since it is a common source of misunderstanding.

Reply:
Actually, I don't think that `ACCOUNT` is considerably better than `USERNAME` (after all, it does not make clear that the context is *not* the Mafiasi account but rather the email address used for registration) – maybe something like `REGISTRATION_EMAIL_LOCAL_PART_PATTERN` (or possibly `REGISTRATION_EMAIL_USERNAME_PATTERN`, though that would only be clean for our specific scenario and not for generic ones)?

Reply:
But `account` is literally the field of the `Mafiasi` entity containing the string that is checked with the regex. Everywhere in the code this property is called `account`, the method where the regex is used is `clean_account` and it is matched against the `account` variable.

Reply:
Ok, you're right – agreed, let's keep it!